app/_layouts/docs-v2.html

```diff
@@ -41,7 +41,9 @@
           {% assign default_edit_link = page.path | replace: "latest", page.kong_version %}
           {% assign edit_link_no_https = page.edit_link | remove: "https://" %}
 
-          {% if page.edit_link and edit_link_no_https == page.edit_link %}
+          {% if page.source_url %}
+            {{ page.source_url }}
```

Comment:
Should this be the edit link? We'd have to do some manipulation of `source_url` but I think it would be better if the experience was consistent

Reply:
The `source_url` is the view link:

https://github.com/Kong/kong/tree/master/kong/pdk/client.lua

vs:

https://github.com/Kong/kong/edit/master/kong/pdk/client.lua

Replace `tree` with `edit` and it's good to go

```suggestion
            {{ page.source_url | replace: "/tree/", "/edit/" }}
```

Reply:
Ah! I didn't even think of that 😬 yep let's do that.